src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -748,6 +751,12 @@ public function getPhinxType($sqlType)
             case 'text':
             case 'json':
                 return static::PHINX_TYPE_TEXT;
+            case 'jsonb':
+                return static::PHINX_TYPE_JSONB;
```

Comment:
Right, but the only difference is in storage, not in structure. Ideally, both would share a common `JSON` type and have a column flag for "is this a binary column?". Similar to how INT columns have an "identity" flag.